src/test/lib/TestConnection.h

```diff
@@ -17,6 +17,8 @@ enum NEW_STREAM_START_TYPE {
     NEW_STREAM_START_ASYNC      // Start asynchronously
 };
 
+#define DEFAULT_SSLKEYLOGFILE_PATH "sslkeylogfile.txt"
```

Comment:
PATH? not FILENAME?

Reply:
I don't know what word to use to describe that it's both path and filename without writing that out and making the name longer